guides/doc-Managing_Content/master.adoc

```diff
@@ -68,7 +65,7 @@ include::common/assembly_importing-kickstart-repositories.adoc[leveloffset=+1]
 endif::[]
 
 endif::[]
```

Comment:
Should this `endif:[]` be moved all the way to the bottom instead? Now it still renders on `foreman-el`.

Reply:
Fixed.